MoltenVK/MoltenVK.xcodeproj/project.pbxproj

```diff
@@ -2695,6 +2695,7 @@
 				MACOSX_DEPLOYMENT_TARGET = 10.15;
 				MARKETING_VERSION = "${CURRENT_PROJECT_VERSION}";
 				MTL_ENABLE_DEBUG_INFO = YES;
+				OTHER_LDFLAGS = "-ld_classic";
```

Comment:
I'm worried that this could cause problems with older versions of Xcode. Unless Bill is willing to drop support for those old versions, there needs to be some sort of version check here.

Reply:
@cdavis5e How "old" are you thinking that might be problematic?

Reply:
I've pushed a change that moves this to be tvOS-specific as I haven't seen the same problem on iOS, and currently MoltenVK doesn't compile for tvOS using Xcode 14 anyway. If there's something that's preferred I'm happy to change this.

Reply:
> @cdavis5e How "old" are you thinking that might be problematic?

Anything prior to Xcode 14, I think. It was either that or 15 that introduced the new new linker. I think older versions of Clang don't understand `-ld_classic` and will actually produce an error diagnostic for the unknown option.

Reply:
> I've pushed a change that moves this to be tvOS-specific as I haven't seen the same problem on iOS, and currently MoltenVK doesn't compile for tvOS using Xcode 14 anyway. If there's something that's preferred I'm happy to change this.

Very well, then.